plugins/gateway/kong/plugin.go

```diff
@@ -78,50 +105,69 @@ func (conf Config) Response(kong *pdk.PDK) {
 			return
 		}
 	}
-	if telemetriesAPI == nil {
-		var tlsOptions *common.ClientTLSOptions
-		if conf.EnableTLS {
-			tlsOptions = &common.ClientTLSOptions{
-				RootCAFileName: common.CACertFile,
-			}
-		}
-		apiClient, err := common.NewTelemetryAPIClient(conf.Host, tlsOptions)
-		if err != nil {
-			_ = kong.Log.Err(fmt.Sprintf("Failed to create new api client: %v", err))
-			return
-		}
-		telemetriesAPI = apiClient
-	}
 	telemetry, err := createTelemetry(kong)
 	if err != nil {
 		_ = kong.Log.Err(fmt.Sprintf("Failed to create telemetry: %v", err))
 		return
 	}
 
-	params := operations.NewPostTelemetryParams().WithBody(telemetry)
-
-	_, err = telemetriesAPI.Operations.PostTelemetry(params)
+	err = apiclarityClient.PostTelemetry(telemetry)
 	if err != nil {
 		_ = kong.Log.Err(fmt.Sprintf("Failed to post telemetry: %v", err))
 		return
 	}
 	_ = kong.Log.Info(fmt.Sprintf("Telemetry has been sent: %v", telemetry))
 }
 
+func processNewDiscoveredAPI(kong *pdk.PDK) error {
+	host, err := getHost(kong)
+	if err != nil {
+		return fmt.Errorf("failed to get routed service for processNewDiscoveredAPI: %v", err)
+	}
+	// Check for newDiscoveredApi
+	if !common.Contains(discoveredApis, host) {
+		appendNewDiscoveredAPI(host)
+		hosts := []string{host}
+		if err := apiclarityClient.PostNewDiscoveredAPIs(hosts); err != nil {
+			return fmt.Errorf("failed to send newDiscoveredApi request: %v", err)
+		}
+		_ = kong.Log.Info("Sent PostNewDiscoveredAPIs with success")
```

Comment:
Should be logged only in Debug mode

Reply:
Ok. But anyway, info or debug are not display on logs. only error appear.

Reply:
I'm just trying to avoid to many log output when everything is working well.